Lib/asyncio/tasks.py

```diff
@@ -634,7 +634,7 @@ def _ensure_future(coro_or_future, *, loop=None):
         loop = events._get_event_loop(stacklevel=4)
     try:
         return loop.create_task(coro_or_future)
-    except RuntimeError: 
+    except RuntimeError:
```

Comment:
```suggestion
    except RuntimeError: 
```

Reply:
Probably this was modified by my IDE without asking me.
Click on `Commit suggestion` to revert. Or let it be!